### PR TITLE
M2k :  Drop Scopy tests and use libm2k instead

### PR DESCRIPTION
**Pull request by AlexandraTrifan**
The M2K production tests will now be run on Raspberry Pi and libm2k is a better option to run the tests, since Scopy is harder to debug and update.

### CODE REVIEW

File: pluto-m2k/config/m2k/m2k_part1.py [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+			return False
+	else:
+		if (value < MAX_LOW_GAIN) and (value > MIN_LOW_GAIN):
+			return True
+		else:
+			return False
+
+def disable_ref_measurement():
+	subprocess.run(["./ref_measure_ctl.sh", "disable"])
+	#extern.start("sshpass -pjig ssh jig@localhost sudo ~/plutosdr-m2k-production-test-V2/ref_measure_ctl.sh disable");
```
Comment (mirceacaprioru) [inline pluto-m2k/config/m2k/m2k_part1.py:61]: is this commented code required ?

Comment (AlexandraTrifan) [reply pluto-m2k/config/m2k/m2k_part1.py:61]: no, I will update the commits to remove the unnecessary comments

File: pluto-m2k/config/m2k/m2k_part1.py [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+	global GAIN_POS_ADC, GAIN_POS_DAC
+	step = 0
+	res = ""
+
+	# Pos dac/adc offset calib with 100mV
+	pws.enableChannel(0, True)
+	pws.pushChannel(0, PWS_POS_FIRST, False)
+	# call some shell script which returns the ADC value
+	value = subprocess.run(["./m2k_power_calib_meas.sh", "V5B pos false"], universal_newlines = False, stdout = subprocess.PIPE)
+	#value = extern.start("sshpass -pjig ssh jig@localhost sudo " + WORKING_DIR + "/m2k_power_calib_meas.sh V5B pos false").trim()
```
Comment (mirceacaprioru) [inline pluto-m2k/config/m2k/m2k_part1.py:228]: again commented out code

**Review by mirceacaprioru — APPROVED** [review]

**Review by adisuciu — APPROVED** [review]

**Review by adisuciu — APPROVED** [review]

**Comment by mhennerich** [comment]
@btogorean good to merge?

**Review by btogorean — APPROVED** [review]
LGTM

**Pull request merged**